### PR TITLE
EFR32MG21.Router

### PR DESCRIPTION
**Pull request by BWFCGUIXT**
Hi Koen,

Not sure if the .MD is correct, should I do it differently please let me know.

### CODE REVIEW

File: docs/devices/EFR32MG21.Router.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| Exposes | linkquality |
+| Picture | ![Custom devices (DiY) EFR32MG21.Router](https://www.zigbee2mqtt.io/images/devices/EFR32MG21.Router.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+
+### LED status
+
+RED LED off = NO Netwrok
```
Comment (Koenkk) [inline docs/devices/EFR32MG21.Router.md:30]: Netwrok -> network

File: docs/devices/EFR32MG21.Router.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+RED LED blink = steering network
```
Comment (Koenkk) [inline docs/devices/EFR32MG21.Router.md:31]: Steering is a difficult term, can you explain it better?

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee-herdsman-converters/pull/4837/files

File: docs/devices/EFR32MG21.Router.md (hunk 1 above)
Comment (BWFCGUIXT) [reply docs/devices/EFR32MG21.Router.md]: Corrected

File: docs/devices/EFR32MG21.Router.md (hunk 2 above)
Comment (BWFCGUIXT) [reply docs/devices/EFR32MG21.Router.md]: Changed the wording

File: docs/devices/EFR32MG21.Router.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| Vendor  | Custom devices (DiY)  |
+| Description | EFR32MG21 router |
+| Exposes | linkquality |
+| Picture | ![Custom devices (DiY) EFR32MG21.Router](https://www.zigbee2mqtt.io/images/devices/EFR32MG21.Router.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+
+### LED status
```
[the remark this answers is not in the document]
Comment (BWFCGUIXT) [reply docs/devices/EFR32MG21.Router.md:28]: Done

**Pull request merged**